### PR TITLE
AnizmTV Premid Extension

### PR DESCRIPTION
**Pull request by Bas1C9**

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof
Dont upload js files
Version 1.0.0 on release
Fix codacy errors

**Comment by Bas1C9** [comment]
Proof
https://imgur.com/nvtZ5Zl
https://imgur.com/B0budQd
https://imgur.com/jqg7uIM

**Review by Bas1C9 — COMMENTED** [review]
Fixed errors

File: websites/A/AnizmTV/presence.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
+const presence = new Presence({
+  clientId: "778715860638367804"
+});
+
+var strings = presence.getStrings({
```
Comment (ririxidev) [inline websites/A/AnizmTV/presence.ts:5]: var -> const, and merge consts

File: websites/A/AnizmTV/presence.ts [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  paused: "presence.playback.paused",
+  browsing: "presence.activity.browsing"
+});
+
+function getTimestamps(
+  videoTime: number,
+  videoDuration: number
+): Array<number> {
+  let startTime = Date.now();
+  let endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
```
Comment (ririxidev) [inline websites/A/AnizmTV/presence.ts:16]: let -> const, merge consts

File: websites/A/AnizmTV/presence.ts [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+presence.on("UpdateData", async () => {
+  
+  const data: PresenceData = {
+    largeImageKey: "anizm"
+  };
+
+  var title = document.querySelector(
+    "html > body > font > main > #pageContent > div > h2 > a"
+  );
+  var episode = document.querySelector(
```
Comment (ririxidev) [inline websites/A/AnizmTV/presence.ts:39]: var -> const

Comment (ririxidev) [inline websites/A/AnizmTV/presence.ts:39]: merge consts

File: websites/A/AnizmTV/presence.ts [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        tags = document.querySelector(
+          "#pageContent > div > h2 > span"
+          );
+        data.state = tags.innerText.split("Aranan: ").slice(1).join(" ");
+      } else if(document.location.pathname.includes("/girisyap")){
+        data.details = "Giriş yapıyor...";
+      } else if(document.location.pathname.includes("/uyeol")){
+        data.details = "Üye oluyor...";
+      } else if(window.location.href.indexOf("?sayfa=") > 1){
+        let pageNum = document.URL.split('?sayfa=')[1].split("#episodes").slice(0).join(" ");
```
Comment (ririxidev) [inline websites/A/AnizmTV/presence.ts:85]: let -> const

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/A/AnizmTV/presence.ts (hunk 3 above)
Comment (Bas1C9) [reply websites/A/AnizmTV/presence.ts]: http://prntscr.com/vmyecj
Bot sees wrong

Comment (ririxidev) [reply websites/A/AnizmTV/presence.ts]: no, it doesn't, please fix codacy errors.

Comment (Bas1C9) [reply websites/A/AnizmTV/presence.ts]: I'm sorry, it's my fault.

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**